.github/workflows/build-and-deploy.yml

```diff
@@ -12,12 +12,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+      - name: Set up QEMU
+        if: runner.os == 'Linux'
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: all
       # Used to host cibuildwheel
-      - uses: actions/setup-python@v3
-      - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.7.0
       - name: Build wheels
-        run: python -m cibuildwheel --output-dir wheelhouse
+        uses: pypa/cibuildwheel@v2.7.0
+        env:
+          CIBW_ARCHS_LINUX: auto aarch64 ppc64le
```

Comment:
We could risk it a little comment for non cibuildwheel experts.
Something like they have in the doc:
```
# configure cibuildwheel to build native archs ('auto'), and some emulated ones
```

Reply:
Sure - I'll put that into the .yml :-)